public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -49,22 +103,107 @@ function ReceiverHealth({ errorsByReceiver }: ReceiverHealthProps) {
   );
 }
 
-interface Props {
-  config: AlertManagerCortexConfig;
-  alertManagerName: string;
-}
-
 const useContactPointsState = (alertManagerName: string) => {
   const contactPointsStateRequest = useUnifiedAlertingSelector((state) => state.contactPointsState);
   const { result: contactPointsState } = (alertManagerName && contactPointsStateRequest) || initialAsyncRequestState;
-  const receivers = contactPointsState?.receivers ?? {};
+  const receivers: ReceiversState = contactPointsState?.receivers ?? {};
   const errorStateAvailable = Object.keys(receivers).length > 0; // this logic can change depending on how we implement this in the BE
   return { contactPointsState, errorStateAvailable };
 };
 
+interface ReceiverItem {
+  name: string;
+  types: string[];
+  provisioned?: boolean;
+}
+
+interface IntegrationItem {
+  lastError?: null | string;
+  lastNotify: string;
+  lastNotifyDuration: string;
+  type: string;
+  sendResolve?: boolean;
+}
+
+type RowTableColumnProps = DynamicTableColumnProps<ReceiverItem>;
+type RowItemTableProps = DynamicTableItemProps<ReceiverItem>;
+
+type RowIntegrationTableColumnProps = DynamicTableColumnProps<IntegrationItem>;
+type RowIntegrationItemTableProps = DynamicTableItemProps<IntegrationItem>;
+
+interface IntegrationsTableProps {
+  integrationTypesState: IntegrationTypesState;
+}
+
+function LastNotify({ lastNotify }: { lastNotify: string }) {
+  return (
+    <Stack alignItems="center">
+      <div>{dateTime(lastNotify).locale('en').fromNow(true)} ago</div>
+      <Icon name="exclamation-triangle" />
+      <Icon name="clock-nine" />
+      <div>{dateTimeFormat(lastNotify, { format: 'YYYY-MM-DD HH:mm:ss' })}</div>
+    </Stack>
+  );
+}
+function IntegrationsTable({ integrationTypesState }: IntegrationsTableProps) {
+  function getIntegrationColumns(): RowIntegrationTableColumnProps[] {
+    return [
+      {
+        id: 'health',
+        label: 'Health',
+        renderCell: ({ data: { lastError } }) => {
+          return <ReceiverHealth errorsByReceiver={lastError ? 1 : 0} />;
+        },
+        size: 0.5,
+      },
+      {
+        id: 'name',
+        label: 'Name',
+        renderCell: ({ data: { type }, id }) => <>{`${type}[${id}]`}</>,
+        size: 1,
+      },
+      {
+        id: 'lastNotify',
+        label: 'Last try to notify',
```

Comment:
nit: I think we can go with a better label here, how about `Last delivery attempt`?